### PR TITLE
office/pdfchain: Update SlackBuild

### PR DESCRIPTION
**Pull request by isaackwy**
Last Friday, I emailed R.S. Ananda Murthy asking him to update this SlackBuild.
He has not replied to me yet.

Many changes to the SlackBuild below.

1. Do not unset variables upon detecting gcc-5. The pdftk SlackBuild (which I am currently maintaining) no longer depends on gcc-5.
2. Add patches originally written by Debian maintainers
3. Change HOMEPAGE
4. Use HTTPS download link
5. Remove sed commands for changing desktop shortcut file (they do not change anything)
6. Other SlackBuild format changes (ex. remove comment on PRINT_PACKAGE_NAME)

### CODE REVIEW

File: office/pdfchain/pdfchain.SlackBuild [hunk 1]
```diff
@@ -107,11 +97,8 @@ make install DESTDIR=$PKG
 find $PKG -print0 | xargs -0 file | grep -e "executable" -e "shared object" | grep ELF \
   | cut -f 1 -d : | xargs strip --strip-unneeded 2> /dev/null || true
 
-sed -i s:'/usr/bin/pdfchain':'pdfchain': $PKG/usr/share/applications/$PRGNAM.desktop
-sed -i s:'/usr/share/pixmaps/pdfchain.png':'pdfchain': $PKG/usr/share/applications/$PRGNAM.desktop
-
 cat $CWD/$PRGNAM.SlackBuild > $PKG/usr/doc/$PRGNAM-$VERSION/$PRGNAM.SlackBuild
-rm $PKG/usr/doc/$PRGNAM-$VERSION/NEWS
+rm $PKG/usr/doc/$PRGNAM-$VERSION/{INSTALL,NEWS}
```
Comment (willysr) [inline office/pdfchain/pdfchain.SlackBuild:101]: Please leave INSTALL out, it's useless when we already have SlackBuild script

Comment (isaackwy) [reply office/pdfchain/pdfchain.SlackBuild]: I now decided not to remove INSTALL.

This sbopkglint output was the reason why I initially removed INSTALL:
```
Exploding /tmp/pdfchain-0.4.4.2-x86_64-2_SBo.tgz to /tmp/SBo/sbopkglint.bHaGFz ...OK
Running pre-doinst test...OK
Installing /tmp/pdfchain-0.4.4.2-x86_64-2_SBo.tgz to /tmp/SBo/sbopkglint.bHaGFz ...OK
Running test: 05-basic-sanity...OK
Running test: 10-docs...
--- generic install instructions found, useless, please remove: usr/doc/pdfchain-0.4.4.2/INSTALL
FAILED
Running test: 15-noarch...OK
Running test: 20-arch...OK
Running test: 25-lafiles...OK
Running test: 30-manpages...OK
Running test: 35-desktop...OK
Running test: 40-newconfig...OK
Running test: 45-doinst...OK
Running test: 50-icons...OK
Running test: 60-usr_info...OK
Running test: 65-python...OK
Running test: 70-tmp_path...OK
Running test: 75-static_libs...OK
Running test: 85-perl...OK
```

**Comment by willysr** [comment]
@sbo-bot: build office/pdfchain

**Comment by willysr** [comment]
merged

**Pull request closed without merging**